### PR TITLE
Bril LLVM

### PR DESCRIPTION
**Pull request by ryan-berger**

### CODE REVIEW

**Comment by oflatt** [comment]
Marking as draft until it runs tests

File: src/main.rs [hunk 1]
```diff
@@ -75,6 +75,7 @@ fn main() {
         output_path: args.output_path,
         optimize_egglog: args.optimize_egglog,
         optimize_brilift: args.optimize_brilift,
+        optimize_brillvm: args.optimize_brilift,
```
Comment (oflatt) [inline src/main.rs:78]: bug here? is the brillvm flag supposed to be different?

File: tests/files.rs [hunk 2]
```diff
@@ -26,6 +26,14 @@ fn generate_tests(glob: &str) -> Vec<Trial> {
                     .unwrap();
             }
 
+            if run.test_type == RunType::CompileBrilLLVM {
```
Comment (oflatt) [inline tests/files.rs:29]: this should probably be the same code path as the if above
you should also generate an args file like the compileBrilift mode for use in nightlies

**Review by oflatt — CHANGES_REQUESTED** [review]
Great work! Some minor things

File: src/main.rs (hunk 1 above)
Comment (ryan-berger) [reply src/main.rs:78]: Yes, definitely a bug, nice catch

**Review by oflatt — APPROVED** [review]
Thanks!

**Pull request merged**